src/components/Questionnaire/QuestionTypes/FileQuestion.tsx

```
@@ -0,0 +1,247 @@
+      {
+        [FILE_UPLOAD_FIELDS.FILE_DATA.key]: value.file_data,
+        [FILE_UPLOAD_FIELDS.NAME.key]: value.name,
+        [FILE_UPLOAD_FIELDS.ORIGINAL_NAME.key]: value.original_name,
+      },
+      questionId,
+      FILE_UPLOAD_FIELDS,
+      index,
+    );
+
+    return [
+      ...errors,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)